experiments/functional/cstor-volume-migration/test.yml

```diff
@@ -0,0 +1,433 @@
+           replace: "{{ cv_name.stdout }}-{{ csp_name.stdout }}"
+          
+       - name: Replace the csp name in cvr yaml
+         replace:
+           path: cvr.yaml
+           regexp: "csp_name"
+           replace: "{{ csp_name.stdout }}"
+
+       - name: Replace the csp uid in cvr yaml
+         replace:
+           path: cvr.yaml
+           regexp: "csp_uid"
+           replace: "{{ csp_uid.stdout }}"       
+
+       - name: Replace the cstor volume name in cvr yaml
+         replace:
+           path: cvr.yaml
+           regexp: "cstor_volume_name"
+           replace: "{{ cv_name.stdout }}"
+
+       - name: Replace the persistent volume name in cvr yaml
+         replace:
+           path: cvr.yaml
+           regexp: "pv_name"
+           replace: "{{ pv.stdout }}"
+
+       - name: Replace the storage class name in cvr yaml
+         replace:
+           path: cvr.yaml
+           regexp: "storage_class_name"
+           replace: "{{ sc_name.stdout }}"
+
+       - name: Apply the cvr yaml to migrate the cvr to new pool
+         shell: kubectl apply -f cvr.yaml
+         args:
+           executable: /bin/bash
+
+       - name: Remove the old cvr to migrate
+         shell: kubectl delete cvr -n {{ operator_ns }} {{ target_cvr }}
+         args:
+           executable: /bin/bash
+         
+       - name: Check if the cvr got removed
+         shell: >
+           kubectl get cvr -n {{ operator_ns }} -l openebs.io/persistent-volume={{ pv.stdout }}
+           -o custom-columns=:.metadata.name --no-headers
+         args:
+           executable: /bin/bash
+         register: new_cvr_list
+         until: "target_cvr not in new_cvr_list.stdout"
+         delay: 5
+         retries: 30
+
+       - name: Check if the CVRs in healthy state
+         shell: >
+           kubectl get cvr -n {{ operator_ns }} -l openebs.io/persistent-volume={{ pv.stdout }}
+           -o custom-columns=:.status.phase --no-headers
+         args: 
+           executable: /bin/bash
+         register: new_cvr_status
+         until: "((new_cvr_status.stdout_lines|unique)|length) == 1 and 'Healthy' in new_cvr_status.stdout"
+         retries: 30
+         delay: 10
+
+       - name: Obtain the new replica id value from cstorvolume
+         shell: >
+           kubectl get cstorvolume {{ cv_name.stdout }} -n {{ operator_ns }}
+           -o custom-columns=:.status.replicaDetails.knownReplicas.{{ replica_id.stdout }} --no-headers
+         args:
+           executable: /bin/bash
+         register: new_rid_value
+         until: "new_rid_value.stdout != rid_value.stdout"
+         retries: 30
+         delay: 10
+
+       - name: Obtain the ReplicationFactor after cvr migration and compare with replication factor obtained before migrate cvr
+         shell: >
+           kubectl get cstorvolume -n {{ operator_ns }} -o jsonpath='{.items[?(@.metadata.name=="{{ cv_name.stdout }}")].spec.replicationFactor}{"\n"}'
```

Comment:
Here we are iterating over all cstorvolume present in the cluster which is not good practice to do since we know cstorvolume we can use directly `kubectl get cstorvolume {{ cv_name.stdout }} -n openebs ....`

Reply:
Same as for cf and drf below tasks